mw-webapp/src/component/checkbox/Checkbox.tsx

```diff
@@ -0,0 +1,42 @@
+interface CheckboxProps {
+  /**
+   * Status flag whether the element is checked or not
+   */
+  checked: boolean;
+  /**
+  * Label's text
+  */
+  label: string;
+  /**
+   * Additional custom class name for the component
```

Comment:
remove useless words which does not help other developers for example you are using the word `Additional`. What does it mean in curent context? For example: Should I think that if you have `additional custom class name` then you should also have `ordinary custom class name`?

Reply:
deleted unnecessary